lib/src/Base/Type/Indices.cxx

```diff
@@ -80,4 +80,38 @@ Indices Indices::complement(const UnsignedInteger n) const
   return result;
 }
 
+/* Return true if the integer n in is the indices */
+Bool Indices::contains(const UnsignedInteger n) const
+{
+  const UnsignedInteger size = getSize();
+  if (!(size > 0)) return false;
+  for (UnsignedInteger i = 0; i < size; ++i)
+    if (operator[](i) == n) return true;
+  return false;
+}
+
+/* Returns the infinite-norm of the indices */
+UnsignedInteger Indices::normInfinite() const
+{
+  const UnsignedInteger size = getSize();
```

Comment:
By the way, use the implementation based on std::max_element, much more efficient and clearer to read (it is the code you removed in `getSobolGroupedIndex`

Reply:
Ok. Fixed at 560c7cff65dae314219b203a994db37cad01d924